float64 anomaly_start_time_in_secs
---
int64 predicted_label 
float64 predicted_proba 
